docs/en/observability/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -308,7 +351,229 @@ NOTE: Starting from **v1.4.0**, if you want to update the Events settings for th
 
 [NOTE]
 ====
-Starting from **v1.4.0**, if you want to update the Events settings for the deployment, it is no longer required to manually delete existing settings before applying the new settings.
+From **v1.4.0** and above, if you want to update the Events settings for the deployment, it is no longer required to manually delete existing settings before applying the new settings.
 
 Due to a https://github.com/hashicorp/terraform-provider-aws/issues/24771[Terraform bug] related to `aws_serverlessapplicationrepository_application`, if you want to delete existing Event parameters you have to set the related `aws_serverlessapplicationrepository_cloudformation_stack.parameters` to a blank space value (`" "`) instead of an empty string (`""`).
 ====
+
+[[aws-serverless-forwarder-direct-deploy]]
+== Deploy Elastic Serverless Forwarder directly
+
+If the customisation options available when deploying via Serverless Application Repository (SAR) are not sufficient, from version `1.6.0` and above you can deploy the Elastic Serverless Forwarder directly to your {aws} Account without using SAR.
+
+// coudl we detail or reframe the customisation options here? Note: have changed most 'publish' references to 'deploy' for consistency
```

Comment:
the direct deployment allows to customise the event sources (ie: triggers) settings one by one, instead of applying some default values.

Something like:
```
If the customisation options available when deploying via Serverless Application Repository (SAR) are not sufficient, from version `1.6.0` and above you can deploy the Elastic Serverless Forwarder directly to your {aws} Account without using SAR.

Deploying the Elastic Serverless Forwarder directly allows you to customise also the settings of the Event Sources for the Inputs (i.e. triggers) one by one.
```